### ISSUE TITLE
Make script work in the Edit Summary

### ISSUE THREAD

**Issue by Benjol** [Benjol/SE-AutoReviewComments#14]
('Imported' from [here](http://stackapps.com/a/2896/876))

It would be great to be able to use this same functionality in the edit summary when making edits. Most of the edits I do fall into a small subset of "categories" and it pains me to have to put in the same summary over and over. 

Adding a category (much like already exists for question-only or answer-only comments) would be a big help beyond just adding the "auto" link next to the edit summary input box.

To take it one step more, since edits may be for more than one thing at a time, a slightly different interface that is checkbox-based, rather than radio buttons would be ideal so that I could combine / chain the edits I've made.

In summary, here's what I am asking for (in order of priority / usefulness):

  1. Adding the dialog link to the Edit Summary input
  2. Adding a category for Edit Summaries
  3. Changing the behavior for edit summaries to allow for adding multiple "comments"

**Comment by Benjol** [comment]
@oliversalzburg, so you can close this too?
